client/consensus/babe/src/lib.rs

```diff
@@ -669,20 +669,26 @@ impl<B, C, E, I, Error, SO> sc_consensus_slots::SimpleSlotWorker<B> for BabeSlot
 
 	fn proposing_remaining_duration(
 		&self,
-		head: &B::Header,
+		parent_head: &B::Header,
 		slot_info: &SlotInfo,
 	) -> Option<std::time::Duration> {
 		let slot_remaining = self.slot_remaining_duration(slot_info);
 
-		let parent_slot = match find_pre_digest::<B>(head) {
+		// If parent is genesis block, we don't require any lenience factor.
+		if parent_head.number().is_zero() {
+			return Some(slot_remaining)
```

Comment:
```suggestion
			return Some(slot_remaining);
```

Reply:
Our style guide actually says that what I'm doing is correct :see_no_evil:

Reply:
Okay 🙈